toolkit/count_spacers.py

```diff
@@ -132,7 +132,7 @@ def _process_fastq_file(self, file, library_sequences):
 		statistics.append('Number of reads where key was not found: {}'.format(key_not_found))
 		statistics.append('Number of reads processed: {}'.format(num_reads))
 		statistics.append('Percentage of guides that matched perfectly: {}'.format(percent_matched))
-		statistics.append('Percentage of undetected guides: {}'.format(percent_no_reads))
+		statistics.append('Percentage of guides detected: {}'.format(percent_no_reads))
```

Comment:
`percent_no_reads` is the number of guides with 0 read counts out of the total number of guides in the library, so I think this was correctly described as 'guides undetected' or not detected, instead of detected.

Reply:
`percent_no_reads` was reporting 100% for the last few libraries so I assumed it was actually the percent of detected guides. However, I looked closer and found there actually appears to be a bug in computing `guides_with_reads`. `numpy.count_nonzero(library_sequences.values())` could be changed to `numpy.count_nonzero(list(library_sequences.values()))`. The value of `numpy.count_nonzero(library_sequences.values())` will always be 1 because `library_sequences.values()` is a 'dict_values' object. Once `guides_with_reads` is off, is throws off computing the correct values for `guides_no_reads` and `percent_no_reads` as well.